frontend/src/scenes/session-recordings/playlist/sessionRecordingsListLogic.ts

```diff
@@ -263,7 +263,8 @@ export const sessionRecordingsListLogic = kea<sessionRecordingsListLogicType>([
             },
         ],
         showFilters: [
-            false,
+            true,
+            { persist: true },
```

Comment:
This is keyed _per playlist_ 🤔 
Do we want that? Or do we rather want it to be global (I think we do). If so - `playerSettingsLogic` (we also have autoplay settings there for example)

Reply:
🤦

Reply:
tinier change...?

![Uploading 2023-07-27 11.24.09.gif…]()